app/src/main/java/com/example/scoutconcordia/Activities/MapsActivity.java

```diff
@@ -168,21 +170,16 @@
     private boolean classToClass = false; //this boolean determines if we are searching from a class in 1 building to a class in another building
 
     // Displays the Map
+    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
```

Comment:
What does this annotation mean? What does it do?

Reply:
I'm not entirely sure why Android Studio brought up that prompt and it essentially just checks that the Android OS is at least running KitKat or higher.